### PR TITLE
Events added

### PR DESCRIPTION
**Pull request by karsasmus**
Thanks for contributing to Postleaf! 💙💚

### Pull Request Summary

Added events to Tag/User/Setting

### Have you...

- [x] Checked that your code adheres to our code guidelines?

---

### Contributor License Agreement

We want Postleaf to remain free and open for everyone to use forever, so please don’t submit any code or content that is licensed or copyrighted without first getting written consent.

Before we accept your pull request, we ask that you agree to the following terms to ensure there are no conflicts with Postleaf’s license now or in the future.

1. I certify that the code I am contributing to this project is completely original or free of any known license or copyright.

2. I agree to allow the developer of Postleaf, A Beautiful Site, LLC, license or relicense my work at their discretion. By submitting this pull request, I agree to forfeit all copyrights, intellectual property rights, and royalties pertaining to my contribution.

3. By entering my name in the space provided below, I agree to all of the terms mentioned above.

Signed: [Marc Apfelbaum]

Date: [2016-09-04]

### CODE REVIEW

File: app/source/classes/Setting.php [hunk 1]
```diff
@@ -15,7 +15,9 @@ class Setting extends Postleaf {
 
     // Adds a setting
     public static function add($name, $value) {
```
Comment (claviska) [inline app/source/classes/Setting.php]: Side note: not sure how I feel about having this wrapper method for update. I guess I added it for consistency, but maybe we should just remove it and use update?

**Comment by claviska** [comment]
It's a long weekend here in the US, so I'm taking some time off to spend with the family. I'll test this out fully tomorrow!

File: app/source/classes/Setting.php (hunk 1 above)
Comment (karsasmus) [inline app/source/classes/Setting.php]: Hmm... good question 😄 

I think we should remove this wrapper and use update only.

**Comment by claviska** [comment]
Post events have been removed for now. I like the syntax of `on` and `off` but I don't really like how these events are being dispatched. I want to take some time to think on it and maybe experiment with some ideas before proceeding with plugin support.

**Pull request closed without merging**